### ISSUE TITLE
[Bug] Strange behavior, why is there a line break?

### ISSUE THREAD

**Issue by flaribbit** [WGUNDERWOOD/tex-fmt#54]
```tex
% remove extra vertical space in reference
\let\OLDthebibliography\thebibliography
\renewcommand\thebibliography[1]{
  \OLDthebibliography{#1}
  \setlength{\itemsep}{0pt}
  \setlength{\parskip}{0pt}
}
```

format the above code will get

```tex
% remove extra vertical space in reference
\let\OLDthebibliography\thebibliography
\renewcommand\thebibliography[1]{
  \OLDthebibliography{#1}
  \setlength{
\itemsep}{0pt}
  \setlength{\parskip}{0pt}
}
```

**Comment by cdesaintguilhem** [comment]
The line break looks like it's triggered by the `\item` from `\itemsep`. This seems to be a bug where the regex doesn't check for a space character after `\item`.

**Comment by WGUNDERWOOD** [comment]
Thanks for catching this! I'm fixing it now. Apologies @cdesaintguilhem I think your PR actually dealt with this case but I probably broke it at some point.

**Comment by WGUNDERWOOD** [comment]
Fixed in 427912e21be07c83b7d59cd24af8987d25232e91.
